src/tlitest/tlitest-run

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cd $(dirname $0)
+echo "RUN_PATH=$PWD"
+
+TEST_DIRS="
+../../lib
+/usr/local/lib
+/lib
+"
+
+for TEST_DIR in $TEST_DIRS; do
+    TEST_FILE=$(find $TEST_DIR -name test_tlog.py|head -1)
+    if [ -n "$TEST_FILE" ]; then
+        break
+    fi
+done
+
+if [ -z "$TEST_FILE" ]; then
+    echo "Cannot find test_tlog.py pytest file"
+    exit 1
+fi
+
+echo "Found $TEST_FILE...changing directory to run pytest"
+cd $(dirname $TEST_FILE)
```

Comment:
There doesn't seem to be a need to change to the directory to run a test with pytest, you can just supply it with a file/dir path.

Reply:
You're right, executing with path should work.  And since this will initially be used from the source tree, I'll switch the entire file to the suggested one liner pytest call.